using-mobile-extensions/adobe-target/README.md

```diff
@@ -75,29 +75,6 @@ To add Target to your app:
        #import ACPIdentity
```

Comment:
Do we need to keep the ACP  info above in this page?

Reply:
Good catch, I'll remove it